debian/xenial/kubeadm/etc/systemd/system/kubelet.service.d/10-kubeadm.conf

```diff
@@ -0,0 +1,8 @@
+[Service]
+Environment="KUBELET_KUBECONFIG_ARGS=--kubeconfig=/etc/kubernetes/kubelet.conf --require-kubeconfig=true"
+Environment="KUBELET_SYSTEM_PODS_ARGS=--pod-manifest-path=/etc/kubernetes/manifests --allow-privileged=true"
+Environment="KUBELET_NETWORK_ARGS=--network-plugin=cni --cni-conf-dir=/etc/cni/net.d --cni-bin-dir=/opt/cni/bin"
```

Comment:
What would I need to get these --cni-conf-dir options? I've been building kubelet off your kubeadm branch for the rpm builds but it doesn't appear to be there.